docs/subnet-pages/subnet-2.md

```diff
@@ -6,29 +6,25 @@ title: "Subnet 2"
 
 ## Name
 
-Machine translation.
+Omron (𝜏, β)
 
 ## Netuid
 2
 
 ## Description
 
-BitTranslate’s process is a synergy between validators and miners. Validators begin by proceeding organic requests or by extracting sample texts in specific languages from the dataset. To guarantee uniqueness, these texts are processed using a Text Generation model.
-
-Once processed, requests are distributed to a network of specialized subnet miners, which are ready for translation. Using advanced AI models, miners ensure the translations are both accurate and of high quality. Validators then step in again to rigorously evaluate these translations using multiple quality estimation techniques:
-
-- **BertScore**: Prioritizing the essence of content, this model rewards translations that resonate deeply with the original material.
-- **TransQuest Technology**: Beyond just words, TransQuest evaluates the depth and context of translations, ensuring they capture the full spectrum of meanings from the source.
-
-Then, miners are rewarded with TAO tokens based on their performance. The mechanisms ensures competitiveness and hence continuous excellence in BitTranslate.
+The purpose of the Omron subnet is to provide verified AI inferences to optimize staking and re-staking yields.
+Omron allows a variety of yield optimization models to be hosted and run against the network while being zk-verified.
+Specifically, this subnet enhances the Bittensor network as a whole by providing external uses for protocols outside the Bittensor ecosystem; thereby amplifying bittensor’s mission throughout a variety of external networks.
```

Comment:
Is this what you mean? See below. If correct, use this below wording:

"In specific, this subnet enhances the Bittensor network by providing utility to protocols and networks outside the Bittensor ecosystem. As a result, this subnet amplifies Bittensor’s mission throughout a variety of such external protocols and networks."